xml/MAIN-SBP-CSP-UpdateInfra.xml

```diff
@@ -119,11 +118,18 @@
 				allow for an easier future expansion. </para>
 			<note>
 				<title>Optional Component</title>
-				<para> The Region Server is an OPTIONAL component. It is however advised to
-					implement an automatic RMT server selection mechanism. This enables for
-					automated registration to give end customers a <quote>launch and use</quote>
-					experience, without the need to manually enter or select a registration server.
-				</para>
+				<para> The Region Server is an OPTIONAL component.
+					It is however advised to implement an automatic RMT server selection mechanism.
+					This enables for automated registration to give end customers a <quote>launch
+					and use</quote> experience, without the need to manually enter or select a
+					registration server. </para>
+				<para>
+					If a cloud provider chooses to not use the automatic RMT selection mechanism in the
+					region server then the cloud provider must replace the a susecloud url resolver for
```

Comment:
Will you accept that the sentence after "susecloud url resolver plugin for zypper" is introduced explains what "susecloud url resolver plugin for zypper" means?

Reply:
It's not just that. :-)

So... product names should use entities wherever possible, for 2 reasons.

1. We rename products sometimes. :-) For instance, CaaSP went through several in my time with SUSE.
2. It means the name will always look the same and match other usages. This eliminates the chances of typos etc.

If **I'm** not sure what you mean then readers may not be.

Are you referring to:

    <!ENTITY cloudreg       "&suse;&reg; Cloud">
    <!ENTITY cloud          "&suse; OpenStack Cloud">

...?

If you're talking about Zypper the packaging tool, then it's a name, so capitalize it. URL is an acronym or initialism depending on who you ask ("you are ell" or "earl") so use block caps.

So instead of:

> the cloud provider must replace the a susecloud url resolver for zypper

It should probably say something like:

"... The cloud provider should replace **the** _&cloud;_ **URL** resolver for **Zypper**..."

Reply:
I think there is confusion which I will try to clear up.

- Zypper has a plugin capability  called "Url Resolver plugins" - https://doc.opensuse.org/projects/libzypp/HEAD/zypp-plugins.html#plugin-url-resolver 
- There is a Zypper Url Resolver plugin called "susecloud"
- The "susecloud" plugin is not related to any products like SUSE Cloud or SUSE OpenStack Cloud.
- @rjschwei in another comment in this PR asked that we use "Url Resolver" to be consistent with https://doc.opensuse.org/projects/libzypp/HEAD/zypp-plugins.html#plugin-url-resolver

I believe I have done this correctly.

Reply:
I was using <command>zypper</command> but have changed to using Zypper because this is referring to the tool and not the command.

Reply:
> I think there is confusion which I will try to clear up.

There was!

>     * Zypper has a plugin capability  called "Url Resolver plugins" - https://doc.opensuse.org/projects/libzypp/HEAD/zypp-plugins.html#plugin-url-resolver

Aha! OK.

>     * There is a Zypper Url Resolver plugin called "susecloud"

OK. What is it or what does it do?

>     * The "susecloud" plugin is not related to any products like SUSE Cloud or SUSE OpenStack Cloud.

Got it. Fine, no problem -- consider my suggestion to use entities withdrawn.

>     * @rjschwei in another comment in this PR asked that we use "Url Resolver" to be consistent with https://doc.opensuse.org/projects/libzypp/HEAD/zypp-plugins.html#plugin-url-resolver

:-( Ouch. This one I do **not** like. Sorry, @rjschwei, but I find that ugly. "Url" implies it's a name or a word; it is 
not, it is an initialism for Universal Resource Locator. "Url" is IMHO wrong and it should always be "URL". "Resolver" is not a proper noun and  does not need to be capitalized.

Consistency is good, but if the openSUSE docs team do not try to be consistent with the SUSE Style Guide, I do not think we need to be consistent with them.

> I believe I have done this correctly.

Fair. I think so too, now that the context is clearer to me.

Reply:
> > ```
> > * There is a Zypper Url Resolver plugin called "susecloud"
> > ```
> 
> 
> OK. What is it or what does it do?

The susecloud zypper plugin is part of the cloud-regionsrv-client package that is installed in a SLES client image that is deployed as an instance within a cloud providers infrastructure.  It provides the URL to the closest RMT servers with a cloud providers infrastructure.

I am treating my portion of the conversation as resolved but will not mark it resolved so @rjschwei can review and comment.